### PR TITLE
6 add bootstrap

### PR DESCRIPTION
**Pull request by AngelFBI**
# Description
- Added bootstrap correctly
- Added simple styles for new task html
These changes complete the issue #6 
 
 # Testing
Go to root and click the Create New Task link to see the simple styles
  
  ## Screenshots 
Captura de Pantalla 2022-09-19 a la(s) 16 41 20

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,9 +8,12 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (oskar-raweng) [inline app/views/layouts/application.html.erb:11]: why is it twice?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,9 +8,12 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (oskar-raweng) [inline app/views/layouts/application.html.erb:9]: 1.

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
```
Comment (oskar-raweng) [inline app/javascript/packs/application.js:8]: great job changing all requires to imports.

**Review by oskar-raweng — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (AngelFBI) [reply app/views/layouts/application.html.erb:11]: I had troubles implementing bootstrap correctly and forgot to delete this line from one of the things I tried

File: app/javascript/packs/application.js (hunk 3 above)
Comment (AngelFBI) [reply app/javascript/packs/application.js:8]: Thanks

**Review by oskar-raweng — APPROVED** [review]
Good job!

**Pull request merged**